### PR TITLE
Update shared-line-appearance.md

### PR DESCRIPTION
**Pull request by ashgupt**

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: Teams/shared-line-appearance.md [hunk 1]
```diff
@@ -68,6 +68,8 @@ Shared line appearance is currently supported by the following apps and devices.
 Managers can add up to 25 delegates, and delegates can have up to 25 managers. There is no limit to the number of delegation relationships that can be created in a tenant. 
  
 If the delegator and delegate are not in the same geographic location, it is up to the PSTN provider to allow caller ID to show up from a different geographic location for a delegated (on behalf of) call. 
+
+Circluar delegation configuration is not permitted, If the Delegated users also have delegations between them, they will only be able to see thier delegation and not the initial delegation
```
Comment (JohanFreelancer9) [inline Teams/shared-line-appearance.md:72]:
```suggestion
Circular delegation configuration is not permitted. If the delegated users also have delegations between them, they will only be able to see their delegation and not the initial delegation.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ashgupt, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**